library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -390,6 +407,47 @@ public void setLoopCount(int loopCount) {
     }
   }
 
+  /**
+   * Register callback to listen to GifDrawable animation end event after specific loop count
+   * set by {@link GifDrawable#setLoopCount(int)}
+   *
+   * Note: Unregister this in onLoadCleared to avoid potential memory leak.
+   * @see GifDrawable#unregisterAnimationCallback(AnimationCallback).
+   *
+   * @param animationCallback Animation callback {@link Animatable2Compat.AnimationCallback}.
+   */
+  @Override
+  public void registerAnimationCallback(@NonNull AnimationCallback animationCallback) {
+    if (animationCallbacks == null) {
+      animationCallbacks = new ArrayList<>();
+    }
+    animationCallbacks.add(animationCallback);
+  }
+
+  /**
+   * Unregister animation callback.
```

Comment:
You can probably avoid javadoc for these methods that isn't specific to their behavior in this particular class. The javadoc for the interface will be shown by default.